include/NovelRT.h

```diff
@@ -120,6 +128,7 @@ namespace NovelRT::Windowing {
 //Engine service types
 #include "NovelRT/Audio/AudioService.h"
 #include "NovelRT/DebugService.h"
+#include "NovelRT/Dotnet/DotnetRuntimeService.h"
```

Comment:
This is just a question really about naming - since its already in `NovelRT::Dotnet` would `RuntimeService` be a better type name? or perhaps `ClrService`? Maybe this is just me, but were calling it `Dotnet` twice here.

I'm also possibly wondering about renaming the namespace to `DotNet` as opposed to `Dotnet` maybe? What do you think?

Reply:
Renamed